### ISSUE TITLE
Textures for the Cherry glass doors and trapdoors

### ISSUE THREAD

**Issue by Kitteh6660** [ReviversMC/modern-glass-doors#16]
Luckily I made the textures for those beforehand so feel free to use those :)

![CherryGlassDoors](https://github.com/ReviversMC/modern-glass-doors/assets/5838739/7c6fcbdb-37d3-43b7-9a1b-f6f04dfa6b73)

[CherryGlassDoorTextures.zip](https://github.com/ReviversMC/modern-glass-doors/files/11715368/CherryGlassDoorTextures.zip)

**Comment by thefirethirteen** [comment]
Yup, yup. I just found out 5 mins ago. I should really direct people to this issue page from now on...

**Comment by thefirethirteen** [comment]
Thanks!

**Comment by thefirethirteen** [comment]
@Kitteh6660 please tell me if you want me to add you to the contributors list. If so, what nickname should I use?

**Comment by Kitteh6660** [comment]
Yes please. Name is Kitteh6660 :)
